samples/msal-react-samples/nextjs-sample/pages/profile.js

```diff
@@ -0,0 +1,58 @@
+import { MsalAuthenticationTemplate, useMsal } from "@azure/msal-react";
+import { loginRequest } from "../src/authConfig";
+import React, { useEffect, useState } from "react";
+import { ProfileData } from "../src/ProfileData";
+import { callMsGraph } from "../src/MsGraphApiCall";
+import Paper from "@material-ui/core/Paper";
+import { Typography } from "@material-ui/core";
+
+const ProfileContent = () => {
+    const { instance, accounts } = useMsal();
+    const [graphData, setGraphData] = useState(null);
+  
+    function requestProfileData() {
+        instance.acquireTokenSilent({
+            ...loginRequest,
+            account: accounts[0],
+            redirectUri: "http://localhost:3000"
+        }).then((response) => {
+            callMsGraph(response.accessToken).then(response => setGraphData(response));
+        });
+    }
+
+    useEffect(() => {
+        requestProfileData();
+    }, []);
+  
+    return (
+        <Paper>
+            { graphData ? <ProfileData graphData={graphData} /> : null }
+        </Paper>
+    );
+};
+
+const ErrorComponent = ({error}) => {
+    return <Typography variant="h6">An Error Occurred: {error.errorCode}</Typography>;
+}
+
+const Loading = () => {
+    return <Typography variant="h6">Authentication in progress...</Typography>
+}
+
+export default function Profile() {
+    const authRequest = {
+        ...loginRequest,
+        redirectUri: "http://localhost:3000"
```

Comment:
This seems to be set in a few places, can it be made the default?

Reply:
Thanks for calling this out! Updated.